Symfony/CS/Tokenizer/Token.php

```diff
@@ -149,6 +145,25 @@ public function equalsAny(array $others, $caseSensitive = true)
     }
 
     /**
+     * A helper method used to find out whether or not a certain input token has to be case-sensitively matched.
+     *
+     * @param bool|bool[] $caseSensitive global case sensitiveness or an array of booleans, whose keys should match
+     *                                   the ones used in $others. If any is missing, the default case-sensitive
+     *                                   comparison is used.
+     * @param int         $key           the key of the token that has to be looked up
+     *
+     * @return bool
+     */
+    public static function isKeyCaseSensitive($caseSensitive, $key)
+    {
+        if (is_array($caseSensitive)) {
+            return isset($caseSensitive[$key]) ? (bool) $caseSensitive[$key] : true;
```

Comment:
@keradus I've added casts to make sure the return val is bool even with silly input (see tests)